app/views/shared/authentication/_email_registration_form.html.erb

```diff
@@ -48,31 +48,46 @@
     <% end %>
     <div class="crayons-field mt-3">
       <%= f.label :profile_image, class: "crayons-field__label" %>
-      <%= f.file_field :profile_image, accept: "image/*", class: "crayons-card crayons-card--secondary p-3 flex items-center flex-1 w-100", required: true %>
+      <%= f.file_field :profile_image, accept: "image/*", class: "crayons-card crayons-card--secondary p-3 flex items-center flex-1 w-100" %>
     </div>
 
     <div class="crayons-field mt-2">
-      <%= f.label :name, class: "crayons-field__label" %>
+      <%= f.label :name, class: "crayons-field__label" do %>
+        Name
+        <span class="crayons-field__required crayons-tooltip" data-tooltip="* - required">*</span>
```

Comment:
I think `data-tooltip` should only be `Required`, not `* - required`.
The screen reader reads the asterisk in the `<span>` already :D

Reply:
I added an `aria-label="Required"` and the screen reader went from:
> Name asterisk asterisk required edit required
to:
> Name required edit required

I think we want to keep the `* -` in the tooltip for the hover effect. I think the latest is the best of both worlds!